internal/snmp/logger.go

```diff
@@ -0,0 +1,26 @@
+package snmp
+
+import (
+	"fmt"
+
+	"github.com/influxdata/telegraf"
+)
+
+type Logger struct {
+	gs Connection
```

Comment:
How about only storing the `host` part here? We don't need anything else, do we?

Reply:
Ok, I see. Could you please add a comment to that option with the information that the "Host" part is only available after instantiating the logger? Alternatively, you can set the internal `host` string of the logger once the host is available, can't you?